sensor:
  serial_num: "D20928"
  resolution: "QVGA"
  fps: 30
  T_cam_offset:
    - [2.22e-16, 2.22e-16, -1.00e+00, 0.00e+00]
    - [-1.00e+00, 0.00e+00, -2.22e-16, 0.00e+00]
    - [0.00e+00, 1.00e+00, 2.22e-16, 1.50e-02]
    - [0.00e+00, 0.00e+00, 0.00e+00, 1.00e+00]
  z_near: 0.001
  z_far: 0.05
  gel_width: 0.01835
  gel_height: 0.02460
  gel_thickness: 0.0046
  gel_min_depth: 0.01954
mm_to_pixel: 17.15
ball_diameter: 6.0
max_depth: 0.02076
visualize:
  normals: True
  points3d: True
  ellipse: True
